backend/mora/service/details.py

```diff
@@ -643,11 +646,13 @@ def convert(start, end, funcid, effect):
         return func
 
     def sort_key(obj):
-        return (
-            obj[mapping.VALIDITY][mapping.FROM],
-            util.get_obj_value(obj, (mapping.PERSON, mapping.NAME)),
```

Comment:
That's an awful lot of code to change, in order to insert what is essentially an `or ''` clause.

Actually, a `default` argument to `get_obj_value` would make this nice and readable.